sklearn/inspection/partial_dependence.py

```diff
@@ -281,22 +289,26 @@ def partial_dependence(estimator, X, features, response_method='auto',
     <sklearn.ensemble.HistGradientBoostingRegressor>`, which do not have an
     ``init`` parameter.
     """
-
     if not (is_classifier(estimator) or is_regressor(estimator)):
         raise ValueError(
-            "'estimator' must be a fitted regressor or classifier.")
+            "'estimator' must be a fitted regressor or classifier."
+        )
 
-    if is_classifier(estimator):
-        if not hasattr(estimator, 'classes_'):
-            raise ValueError(
-                "'estimator' parameter must be a fitted estimator"
-            )
-        if isinstance(estimator.classes_[0], np.ndarray):
-            raise ValueError(
-                'Multiclass-multioutput estimators are not supported'
-            )
+    if isinstance(estimator, Pipeline):
+        for est in estimator:
+            if est not in (None, 'drop'):
+                check_is_fitted(est)
```

Comment:
Agreed. I recall some discussion there: https://github.com/scikit-learn/scikit-learn/pull/14545#discussion_r310692191

Reply:
Since that it might be due to the design of `Pipeline`, we might also want to avoid implementing it, thought.

Reply:
I can't say this is elegant. I think it at least needs a comment "Assuming Pipeline is the only estimator that does not store a new attribute" Indeed, this can be removed after `n_features_in_` is required, can it not??

Should we be checking *every* step with `check_is_fitted`, or should we just be checking the last step??? I suspect the latter.